internal/auth/auth.go

```diff
@@ -116,6 +116,29 @@ func WaitUntilUserLogsIn(ctx context.Context, httpClient *http.Client, state Sta
 	}
 }
 
+var RequiredScopes = []string{
```

Comment:
IMO, doesn't make sense to have these scopes as a standalone file. Decided to pull-in to the `auth.go` file.